tests/virt/device_test.py

```diff
@@ -536,6 +536,30 @@ class TestHotplug(TestCaseBase):
 ''' % {'sd_id': SD_ID,
        'vol_id': VOL_ID,
        'img_id': IMG_ID}
+    BLOCK_DISK_HOTPLUG = '''<?xml version='1.0' encoding='UTF-8'?>
+<hotplug>
+  <devices>
+    <disk snapshot="no" type="block" device="lun" sgio="unfiltered">
+      <target dev="sda" bus="scsi"/>
+      <source dev="/dev/mapper/lun1">
+        <reservations managed="yes"/>
+        <seclabel model="dac" type="none" relabel="no"/>
+      </source>
+      <driver name="qemu" io="native" type="raw" error_policy="stop"
+            cache="none"/>
+      <alias name="name1"/>
+      <address bus="0" controller="0" unit="1" type="drive" target="0"/>
+    </disk>
+  </devices>
+  <metadata xmlns:ovirt-vm="http://ovirt.org/vm/1.0">
+    <ovirt-vm:vm>
+      <ovirt-vm:device devtype="disk" name="sda">
+        <ovirt-vm:GUID>lun-uuid</ovirt-vm:GUID>
```

Comment:
Based on the <source>, the value here should be "lun1".

Reply:
Ack